docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/dockerhub/index.md

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_label: Docker Hub
+description: How to configure Docker Hub webhooks with Kargo for automated artifact discovery.
+---
+
+# Docker Hub Webhook Receiver
+
+The Docker Hub Webhook Receiver responds to `push` events originating from
+Docker Hub repositories.
+
+In response to a `push` event, the receiver "refreshes" `Warehouse` resources
+subscribed to the corresponding Docker Hub repository.
+
+:::info
+"Refreshing" a `Warehouse` means enqueuing it for immediate reconciliation by
+the Kargo controller, which will attempt to discover new artifacts from all
+subscribed repositories.
+:::
+
+:::note
+Docker Hub webhook payloads are **not** signed. Kargo uses a static token for
+basic validation.
+:::
+
+## Configuring the Receiver
+
+To enable webhook support for Docker Hub, you must configure a Kubernetes
+`Secret` and reference it in your `ProjectConfig`.
+
+The `Secret` must include a `secret` key under `stringData`. This value is used
+to generate a unique, hard-to-guess URL for the webhook receiver, providing
+basic protection against unauthorized requests.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dh-wh-secret
+  namespace: kargo-demo
+stringData:
+  secret: <your-static-token-here>
+```
+
+Next, reference this `Secret` in your `ProjectConfig`:
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: ProjectConfig
+metadata:
+  name: kargo-demo
+  namespace: kargo-demo
+spec:
+  webhookReceivers:
+    - name: dockerhub-receiver
+      dockerhub:
+        secretRef:
+          name: dh-wh-secret
+```
```

Comment:
Just put these together in one chunk of YAML. It's a tidier, more comprehensive view of what's being done. _This is how the GitHub page is doing it._